evergreen.yml

```diff
@@ -514,15 +514,15 @@ functions:
         working_dir: src/genny
         script: |
           set -eu
-          versions=("master" "8.0" "7.3" "7.0" "6.0" "5.0")
+          versions=("master" "8.0" "7.0" "6.0" "5.0")
 
           # Move original files to backup locations
           for version in $versions; do
             mv evergreen/system_perf/$version/genny_tasks.yml evergreen/system_perf/$version/genny_tasks.bak.yml
           done
 
           # Regenerate the genny task files
-          ./run-genny auto-tasks-local --dsi-path modules/dsi --private-workloads-path modules/PrivateWorkloads
+          ./run-genny auto-tasks-local --dsi-path ../modules/dsi --private-workloads-path ../modules/PrivateWorkloads
```

Comment:
Moved the modules out of the genny repo, otherwise the private workloads gets picked up twice due to the symlink.